### PR TITLE
Alert users if sign out fails.

### PR DESCRIPTION
**Pull request by Elland**
Better than silent failure. Currently investigating how to avoid it altogether.

### CODE REVIEW

File: Toshi/Supporting Files/AppDelegate.m [hunk 1]
```diff
@@ -141,8 +141,8 @@ - (void)userDidSignOut {
 
         exit(0);
     } failure:^(NSError *error) {
-        // alert user
-        NSLog(@"Error attempting to unregister text secure.");
```
Comment (yuliaveres) [inline Toshi/Supporting Files/AppDelegate.m]: i think we should leave log no ?

Comment (Elland) [inline Toshi/Supporting Files/AppDelegate.m]: Why? 🤔

Comment (yuliaveres) [inline Toshi/Supporting Files/AppDelegate.m]: it is handy to see important issues logs in debugger, as for me

File: Toshi/Supporting Files/AppDelegate.m [hunk 2]
```diff
@@ -141,8 +141,8 @@ - (void)userDidSignOut {
...
+        UIAlertController *alert = [UIAlertController alertControllerWithTitle:@"Could not sign out" message:@"Error attempting to unregister from chat service. Ours engineers are looking into it." preferredStyle:UIAlertControllerStyleAlert];
```
Comment (yuliaveres) [inline Toshi/Supporting Files/AppDelegate.m]: don't you need any action here ? 😄

Comment (yuliaveres) [inline Toshi/Supporting Files/AppDelegate.m]: it is not dismissable prolly 😄

Comment (Elland) [inline Toshi/Supporting Files/AppDelegate.m]: 🤦🏽‍♂️ used to the SweetUIKit one!

Comment (yuliaveres) [inline Toshi/Supporting Files/AppDelegate.m]: btw, you can use handy UIAlertController dismissableAlert ...

**Comment by Elland** [comment]
Changes are up. 🤞🏽

**Pull request merged**